src/pages/Authorizing.vue

```diff
@@ -25,6 +26,7 @@
   }
 
   i {
+    font-size: 1.99999999em;
```

Comment:
Instead of using this, you also add a `.fa-fw` class to the icon (makes icon fixed width, square), I just learned about this helper class recently https://fontawesome.com/how-to-use/on-the-web/styling/fixed-width-icons

Reply:
Doesn't seem to work though. It's still set to `28x29`.

![Screenshot 2019-12-14 at 10 15 22 PM](https://user-images.githubusercontent.com/984069/70854680-48f7af00-1ebf-11ea-8cf0-5ff60dcab93f.png)

Reply:
ohh, I suppose it's fixed **width** but not height necessarily